ceph.spec.in

```diff
@@ -3,6 +3,7 @@
 %bcond_with tests
 %bcond_without tcmalloc
 %bcond_without libs_compat
+%bcond_with librocksdb
```

Comment:
I think we want `%bcond_without` here, so that we default to building with rocksdb? Otherwise this change is a behavior change, right?